paper.md

```diff
@@ -168,17 +168,17 @@ Since then, DICOM conversion to NIfTI files contained within a BIDS hierarchy ha
 
 # Overview of HeuDiConv functionality
 
-HeuDiConv was initially developed to implement common for every lab logic (groupping DICOMs, extraction of metadata, conversion of individual sequences, populating standard BIDS files, etc.) while allowing individual groups to customize **how** files should be organized and named while driving custom decisions by the conventions and desires of those individual groups.
+HeuDiConv was initially developed to implement logic commonly used across labs (grouping DICOMs, extracting metadata, converting individual sequences, populating standard BIDS files, etc.) while allowing individual groups to customize **how** files should be organized and named while driving custom decisions through the conventions and desires of those individual groups.
 Such decision making is implemented in *HeuDiConv heuristics*, which are implemented as Python modules following some minimalistic specified interfaces documented in HeuDiConv documentation (https://heudiconv.readthedocs.io/en/latest/heuristics.html).
-HeuDiConv, if instructed to operate in BIDS mode (`--bids` flag) after heuristic provides base naming instructions, takes care about correct placement of files in the hierarchy, naming of multi-echo and other split files, etc.
+HeuDiConv, if instructed to operate in BIDS mode (`--bids` flag) with a heuristic providing base naming instructions, organizes the files in the hierarchy defined by the BIDS standard. It also ensures files are named according to the BIDS specifications, including complex split files such as those associated with multi-echo sequences.
```

Comment:
(Un)fortunately HeuDiConv doesn't really place files into hierarchy -- it remains for a heuristic to do that but indeed HeuDiConv provides some placeholders ... which we did not document much, so submitted https://github.com/nipy/heudiconv/pull/681  . So, what about

```suggestion
HeuDiConv, if instructed to operate in BIDS mode (`--bids` flag) with a heuristic providing base naming instructions, and helpers to organize the files in the hierarchy defined by the BIDS standard.
It also ensures files are named according to the BIDS specifications, including complex composite recordings such as those associated with multi-echo sequences.
```